.github/workflows/log-viewer-docs.yaml

```diff
@@ -2,7 +2,19 @@ name: "docs"
 
 on:
   pull_request:
+    paths:
+        - docs/**/*
+        - .github/workflows/log-viewer-docs.yaml
+        - .gitmodules
+        - taskfile.yaml
+        - tools/**/*
```

Comment:
* All strings should be quoted in our YAML files.
* `**/*` is equivalent to `**` in [GitHub's workflow syntax](https://docs.github.com/en/actions/reference/workflows-and-actions/workflow-syntax#filter-pattern-cheat-sheet).

Reply:
Maybe he was talking about taskfiles?

Reply:
ah, I proposed not specifically the **/* syntax but just a list of files to be included on the hook so this workflow doesn't get triggered for every file modified